bin/sirius-script-rabpm-mngr.py

```diff
@@ -18,33 +18,42 @@ def get_hostname(rack):
         raise ValueError("Invalid rack number")
 
 
+def get_mch_hostname(rack):
+    if rack == "21":
+        return "ia-20rabpmtl-co-cratectrl"
+    elif rack.isdigit() and 1 <= int(rack) <= 20:
+        return f"ia-{rack.zfill(2)}rabpm-co-cratectrl"
+    else:
+        raise ValueError("Invalid rack number")
```

Comment:
Could we have a common function for the rack numbering part? And have `get_cpu/mch_hostname` just call it passing the specific suffix (`iocsrv` and `cratectrl`)